### PR TITLE
Add ability to uninstall plugins

### PR DESCRIPTION
**Pull request by fatkodima**
This is a reworked #5473

### CODE REVIEW

**Comment by deivid-rodriguez** [comment]
@fatkodima Thanks for all the fixes, including this one! :)

Before I have a look, if you used the code in the original #5473 PR as a starting point, would you mind adding [coauthorship information](https://help.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors#creating-co-authored-commits-on-the-command-line) to the commit, so that @Shekharrajak is properly credited?

Thanks!

**Comment by fatkodima** [comment]
Added `Co-authored-by`.

**Comment by Shekharrajak** [comment]
You can also cherry-pick the commit (or use the same branch after the rebase)and do the changed top of that.

File: lib/bundler/cli/plugin.rb [hunk 1]
```diff
@@ -23,6 +23,16 @@ def install(*plugins)
       Bundler::Plugin.install(plugins, options)
     end
 
+    desc "uninstall PLUGINS", "Uninstall the plugins"
+    long_desc <<-D
+      Uninstall given list of plugins. To uninstall all the plugins, use -all option.
```
Comment (jethrodaniel) [inline lib/bundler/cli/plugin.rb]: should be `--all` here

**Pull request closed without merging**